migrations/20201114170749-create-user.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('Users', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      fullName: {
+        type: Sequelize.STRING,
+        allowNull: false
+      },
```

Comment:
According to the design, seem like FE will call some api with `firstName` and `lastName`. Should we consider splitting this attribute `fullName` into `firstName` and `lastName` ? @dothanhlam97 @ndkhaivn 

![image](https://user-images.githubusercontent.com/43344276/99179993-5f585b80-275d-11eb-885b-20af146ca4df.png)